conn2res/connectivity.py

```diff
@@ -16,19 +16,37 @@
 
 class Conn:
     """
-    Class that represents a connectivity matrix representing either weighted
-    or unweighted connectivity data
+    Class that represents a weighted or unweighted network using connectivity
+    data
 
-    Attributes
-    ----------
-    # TODO
+    Note that the diagonal of the matrix is set to zero as well as
+    inf and nan values are replaced to zero. We also makes sure that all nodes
+    are connected to the rest of the network (otherwise the network is
+    reduced to the largest component)
 
-    Methods
+    Parameters
     ----------
-    # TODO
+    w : (N, N, M) numpy.ndarray, optional
+        connectivity matrix (source, target) passed directly instead of
+        being loaded from disc
+        N: number of nodes in the network
+        M: number of subjects
+        subj_id is ignored if array is 2-dimensional (N, N)
+    filename : str, optional
```

Comment:
This is something that we need to fix but I am still not sure what is the best way to do it. Because right now the data is not available in the repo. We could either upload the data to the repo, or every time this method is called without either `filename` or `w`, we could directly download the data from a data repo (see for instance my answer to issue [#9](https://github.com/netneurolab/conn2res/issues/9). You do not have to fix it right away, but something to have in mind!

Reply:
I would suggest to:

- add the data to the repo manually or
- raise an error if filename is not provided and data is not available and point them to the README where I would add a short description how to get the data from zenodo (something you have already done in your other repo and your reply)

Automatically downloading the data seems a bit of an overkill to me and you might run into further troubles if e.g., downloading is not succesful for some reason (e.g. due to intermittent internet connection)

Reply:
I will see what is the best option for this. I am not sure yet!